spec/fixtures/order_statuses.rb

```diff
@@ -0,0 +1,2 @@
+new_order_status:
+  name: New
```

Comment:
Code Climate flagged this. Guessing it was meant to be `.yml` but is it being used?